corehq/apps/locations/bulk_management.py

```diff
@@ -327,13 +327,16 @@ def validate_and_parse_stubs_from_excel(self):
         actual = set(type_sheet_reader.headers)
         expected = set(LOCATION_TYPE_SHEET_HEADERS.values())
         if actual != expected:
-            raise LocationExcelSheetError(
-                _(u"'types' sheet should contain exactly '{expected}' as the sheet headers. "
-                  "'{missing}' are missing")
-                .format(
-                    expected=", ".join(expected),
-                    missing=", ".join(expected - actual))
+            message = (u"'types' sheet should contain exactly '{expected}' as the sheet headers. "
+                       "'{missing}' are missing")
+            if actual - expected:
+                message = message + " '{extra}' are not recognized"
+            message = message.format(
+                expected=", ".join(expected),
+                missing=", ".join(expected - actual),
+                extra=", ".join(actual - expected),
             )
+            raise LocationExcelSheetError(_(message))
```

Comment:
Thanks. I updated it to use lazy strings that way makemessages actually picks up these strings to translate